build-system/tasks/presubmit-checks.js

```diff
@@ -509,12 +514,6 @@ var forbidden3pTerms = {
   // usage in babel's external helpers that is in a code path that we do
   // not use.
   '\\.then\\((?!callNext)': ThreePTermsMessage,
-  'Math\\.sign': ThreePTermsMessage,
-  'Object\\.assign': {
```

Comment:
I suppose these are not needed anymore because we do have those polyfills in [integration.js](https://github.com/ampproject/amphtml/blob/master/3p/integration.js#L25)
@cramforce could you confirm?